packages/features/eventtypes/components/CreateEventTypeDialog.tsx

```diff
@@ -293,12 +294,12 @@ export default function CreateEventTypeDialog({
               </div>
             )}
           </div>
-          <div className="mt-10 flex justify-end gap-x-2">
+          <DialogFooter showDivider>
```

Comment:
Was added the DialogFooter component to set the showDivider as true in this case, with this we can achieve the expected behavior for this modal
![244419451-cae6c7d7-314d-495f-843e-b91d8172b0ac](https://github.com/calcom/cal.com/assets/121884634/28418b92-58a2-4a2a-99a8-5c8bf14ec1b4)